ui/client/components/graph/node-modal/editors/Validators.ts

```diff
@@ -0,0 +1,35 @@
+import _ from "lodash"
+
+export enum validatorType {
+  MANDATORY_VALUE_VALIDATOR = "MandatoryValueValidator",
+  ERROR_VALIDATOR = "ErrorValidator",
+}
+
+export type Validator = {
+  isValid: (...args: any[]) => boolean
```

Comment:
are possible args really unknown?

Reply:
Values which can be passed depends on specific valdiator. I would leave like that right now.